controllers/imagejob/imagejob_controller.go

```diff
@@ -18,18 +15,36 @@ package imagejob
 
 import (
 	"context"
+	"log"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
 
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller"
 	"sigs.k8s.io/controller-runtime/pkg/handler"
-	"sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
 	eraserv1alpha1 "github.com/Azure/eraser/api/v1alpha1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	dockerPath     = "/var/run/dockershim.sock"
```

Comment:
@cpuguy83 should docker and crio path be `/run/..` also?

Reply:
Typically `/var/run` is a symlink to `/run`.